core/iwasm/aot/aot_loader.c

```diff
@@ -1409,6 +1424,13 @@ load_import_table_list(const uint8 **p_buf, const uint8 *buf_end,
             }
         }
 #endif
+
+        /* TBC: sync up with AOT_CURRENT_VERSION in config.h */
+        /* v3 doesn't emit names */
+        if (module->package_version >= 4) {
```

Comment:
Seems that the import table in the older aot file isn't used at all or developer doesn't generate the import table? Maybe we can remove the line?

Reply:
I'm not certain. I'll add a TODO note here, maintain the 'if' control, and run a full compatible test later.